README.md

```diff
@@ -35,26 +35,29 @@ The application is an HTTP server with the
 In the background, the program periodically retrieves data from the Razza API
     and updates the feed.
 By default, this update occurs every 5 seconds.
- 
+
 There are a couple flags that can be passed to the binary:
 
 - `--port <int>`: the port to bind the HTTP server to (default `8080`)
- 
+
 - `--timeout_period <duration>`:
-    	the maximum duration to wait for a response from the source API (default 5s)
+        the maximum duration to wait for a response from the source API (default 5s)
 
 - `--update_period <duration>`:
-    	how often to update the feed (default 5s).
+        how often to update the feed (default 5s).
     Remember that the more frequently you update, the more stress you place
     on the source API, so be nice.
 
 - `--use_http_source_api`
     use the HTTP source API instead of the default gRPC API.
-    
+
+- `--use_panynj_api`:
```

Comment:
In your PR introduction you mention that this alternative API is missing some data compared to the Razza API? What data is missing? (It’s tricky for me to figure out because I’m reading this on a phone!)

Reply:
I think I phrased that poorly - what I meant there was just that the Razza API was a more complete representation of the PATH system, since it includes things like station and route information (whereas with the new API, this has to be manually mapped via `gtfsmapping`).

The Razza API also could theoretically be extended to provide alerts in the future, since these are, to my knowledge, only available in the app via the SQLite DB + some other API calls. I did some research into this previously, and my conclusion was that there isn't a great way to get alerts via just API calls: https://github.com/jamespfennell/path-train-gtfs-realtime/issues/2. This could of course change in the future.

This is a long-winded way of saying that the new API should have feature parity with the Razza API for getting arrival time estimates, but is more limited for future features such as alerts.

Reply:
Thanks for the comment, it makes sense! Hard coding the static data seems fine because it changes so rarely. I do wonder if there’s a way to use the GTFS static data here…but could investigate that later :)